server.js

```diff
@@ -19,12 +13,69 @@ const app = express();
 app.use(cors());
 app.use(express.json());
 
-// Start defining your routes here
+// listed endpoints here, had to removed it since it crashed app - new try when code is done
+
+
+// Start Route
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.send(" Go to /endpoints to see available endpoints ");
+});
+
+//shows the available endpoints
+app.get("/endpoints", (req, res) => {
+  res.send(listEndpoints(app))
+});
+
+
+app.get("/books", (req, res) => {
+  const { author, title  } = req.query; 
+  let filteredBooks = booksData;
+
+  if (author) {
+    filteredBooks = filteredBooks
+    .filter((item) => item.authors.toLocaleLowerCase()
+    .includes(author.toLocaleLowerCase()))
+  };
+  if (title) {
+    filteredBooks = filteredBooks
+    .filter((item) => item.title.toLocaleLowerCase()
+    .includes(title.toLocaleLowerCase()))
```

Comment:
were you going to use these as params or queries? :) they are not listed in the endpoints